documentation/Learn-Beton/Frequently-Asked-Questions.md

```diff
@@ -161,6 +161,67 @@ These events will set a specific tag. Now when the player starts the conversatio
 should check (using the `globaltag` condition) which tag is currently set and give the player different quests based on that.
 Of course, the static folder event also needs to remove the the current tag before setting a new one.
 
+## Global Quests (all players work together)
+
+There is no easy way to do this (yet). Additionally, every use case differs. Let's assume you have some sort of event
+on your server where your player's need to fish 100 salmons. The quest package is only installed during the event.
+
+Create an objective that is immediately fired upon the first interaction (this means setting the amount to one for most objectives).
+That objective must be `persistent` so it restarts immediately upon completion. It also has to be `global` so every
+player will receive it upon joining the server.
+```YAML
+# fish a salmon to progress the global quest
+gQuest: fish SALMON 1 events:gQuestProgress global persistent
+```
+The objective would trigger a folder event that increases a `globalpoint` variable by one and tries to run the
+events that are fired upon completion. That globalpoint variable tracks the players combined progress.
+The "completion events" must be limited by a `globalpoint` condition that checks whether the `globalpoint` variable has
+reached a certain value.
+
+=== "events.yml"
+    ```YAML
+    # 1. increase the global variable 2. wait one tick for the change to process 3. attempt to run the completion events
+    gQuestProgress: folder gQuestIncrementCounter,gQuestCheckCompletion period:1ticks
```

Comment:
instead should be:

gQuestProgress: folder gQuestIncrementCounter,gQuestCheckCompletion period:1 ticks